src/nl/documentatie/10.woocommerce-v5.0.md

```diff
@@ -0,0 +1,329 @@
+
+Order modus inschakelen om hele orders naar MyParcel te exporteren.
+
+##### Concept zendingen
+
+Concept zendingen inschakelen.
+
+##### Direct verwerken
+
+Orders direct naar MyParcel exporteren zodra ze zijn aangemaakt en betaald.
+
+##### Deel klantinformatie
+
+Deel klantinformatie met MyParcel.
+
+##### Track & Trace in email
+
+Track & Trace in e-mail inschakelen.
+Zet "Track & Trace in e-mail" aan om de barcode en Track & Trace link mee te sturen in de verzendbevestiging naar je klanten. De Track & Trace code wordt dan toegevoegd aan de "Bestelling afgerond en/of verzonden"-mail, die wordt verzonden vanuit WooCommerce op het moment dat een bestelling de status "Afgerond" krijgt. We adviseren de track en trace-optie in je MyParcel-account uit te zetten, als je gebruik maakt van deze optie. Anders krijgen je klanten meerdere mails over hun verzonden pakket.
+
+::: note
+Let op! Zorg dat "Direct verwerken" is ingeschakeld, wanneer je deze optie gebruikt in combinatie met de Track & Trace in email optie. Anders wordt de Track & Trace code niet opgenomen in de email naar je klant, omdat er op het moment van verzenden nog geen barcode is gecreëerd.
+:::
+
+##### Track & Trace in account
+
+Track & Trace in account inschakelen. Als je klanten een account kunnen aanmaken in je webshop, kun je met er met deze optie voor zorgen dat de link van de barcode van de bestelling in het account wordt getoond.
+
+##### Barcode in notitie
+
+Barcode in notitie inschakelen
+
+##### Barcode in notitie titel
+
+Je kunt hier de tekst veranderen dat voor de barcode van de zending komt te staan in de notities van de bestelling. Dit is de tekst die voor de barcode komt te staan in de notities.
+
+::: note
+Let op: maak je gebruik van een koppeling met Effect Connect om Bol.com-orders in jouw WooCommerce-omgeving te krijgen? Laat de standaard waarde Tracking code: staan om de barcode automatisch terug te koppelen naar Bol.com.
+:::
+
+<hr/>
+<br/>
+
+### Bestellingen
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_b_nl.jpeg" alt="woocommerce plugin bestellingen" />
+
+#### Automatisering orderstatus
+
+##### Klantadres opslaan in adresboek
+
+Klantadressen van bestellingen opslaan in je MyParcel adresboek
+
+##### Bestelstatus bij label aanmaken
+
+Selecteer de order status die gebruikt moet worden wanneer een label wordt aangemaakt
+
+##### Bestelstatus bij label scannen
+
+Selecteer de order status die gebruikt moet worden wanneer een label is gescand
+
+##### Bestelstatus bij bezorging
+
+Selecteer de order status die gebruikt moet worden wanneer een zending is afgeleverd.
+
+##### Bestelstatus mail
+
+Selecteer de order status die gebruikt moet worden voor de Track & Trace e-mail.
+
+###### Stuur notificatie na
+
+Selecteer het aantal dagen na welke de Track & Trace e-mail verstuurd moet worden
+
+##### Digitale postzegel bestelstatus
+
+Selecteer de order status die gebruikt moet worden voor digitale postzegels.
+
+### Standaard gewichten
```

Comment:
Dit is nu een nieuw kopje net als "Algemeen" en "Bestellingen", ik denk dat het duidelijker is als die kopjes overeen komen met de tabs op de instellingen pagina